app/views/layouts/exercise_inputs/layouts/_input_kids.html.erb

```diff
@@ -3,7 +3,7 @@
     <div <%= turbolinks_enable_for exercise %>>
 
       <div class="mu-kids-landscape-support container-fluid">
-        <img src="/kibi_animated.svg"/>
+        <img src="/character/kibi/jump.svg"/>
```

Comment:
Should we use   `<img id="mu-kids-character-context"/>` here too?

Reply:
Hmmm, I'm assuming in clips that canvas is a single img element, but maybe instead of passing `$(...)[0]` I should just pass `$(...)` and use `.attr('src', ...)` to set the img source for all results?

Reply:
Exactly

Reply:
Done